### PR TITLE
Added response and diagnosis words as in fep_reset page

### PR DESCRIPTION
**Pull request by richardjedgar**
Minor changes to the DEA Sequencer Reset anomaly page response and diagnosis words, to match the fep_reset page.

Dunno why the diffs don't show up?

### CODE REVIEW

**Comment by Gregg140** [comment]
You might have to change the branch you are looking at as I cloned from 
a branch and not Master.

Try fep_reset_may2017

**Comment by jzuhone** [comment]
@richardjedgar it looks like you added a new file to the top-level directory in `acis_anomaly_pages` when you should have edited the `dea_seq_reset.rst` in the `source` subdirectory. 

Probably the best thing to do is make the edits on that latter file, delete the new file you made (`git rm`), and push up the changes to this branch and PR. Everything else here is fine.

**Comment by richardjedgar** [comment]
Ah. Thanks. Hopefully it's fixed now.

File: source/dea_seq_reset.rst [hunk 1]
```diff
@@ -29,12 +29,35 @@ How is this Anomaly Diagnosed?
...
-  (1DPICACU and/or 1DPICBCU) ~0.2-0.3 A. The drop in two input currents depends on which side is
-  running which FEPs.
+  (1DPICACU and/or 1DPICBCU) ~0.2-0.3 A. The drop in two input currents depends on which FEPs
+  were active (DPA Side A runs FEPs 0, 1, and 2; DPA side B runs FEPs 3, 4, and 5.)
 * Within a major frame, the 1STAT1ST bilevel will toggle off (science idle).
 * DEA Housekeeping will show sharp drops in the temperatures of FEP 0 and/or FEP 1, if they are
   running.
 
+If only some of the FEPs were halted and the science run continues, this might be a
+case of :ref:`fep_reset`.
```
Comment (jzuhone) [inline source/dea_seq_reset.rst]: Change :ref:`fep_reset` to :doc:`../fep_reset` here to fix the broken link.

**Review by jzuhone — CHANGES_REQUESTED** [review]

File: source/dea_seq_reset.rst (hunk 1 above)
Comment (richardjedgar) [reply source/dea_seq_reset.rst]: no backticks?

Comment (jzuhone) [reply source/dea_seq_reset.rst]: sorry, yes, backticks around the ../fep_reset part. unfortunately it doesn't render correctly in here.

**Review by jzuhone — APPROVED** [review]

**Comment by richardjedgar** [comment]
@cegrant approved by e-mail.

**Comment by richardjedgar** [comment]
Paul approved by e-mail

**Comment by Gregg140** [comment]
Looks good to me.

**Pull request merged**